docs/LayoutSpecs.json

```diff
@@ -2,7 +2,7 @@
   "elements": [
     {
       "text": "List",
-      "iconName": "Category.List.svg",
+      "iconName": "/src/resources/icons/Category.List.svg",
```

Comment:
Thanks @yeexinc, since we're renaming stuff (i.e. breaking the contract), can we rename `iconName` to `iconUrl`?